criu/util.c

```diff
@@ -334,7 +334,11 @@ void close_proc(void)
 
 int set_proc_fd(int fd)
 {
-	if (install_service_fd(PROC_FD_OFF, dup(fd)) < 0)
+	int _fd = dup(fd);
+
+	if (_fd < 0)
+		return -1;
```

Comment:
Please split _fd initialization and declaration. It does not look readable when we check error of the call from variable declarations.

Also please add proper error message here.